src/indexer/stamper.rs

```diff
@@ -78,5 +110,28 @@ mod test {
 
         assert_eq!(stamper.stamp(), 10u64);
         assert_eq!(stamper_clone.stamp(), 11u64);
+
```

Comment:
Once you use Range you can simply this unit test and `assert_eq!` on the range itself, and remove the other test.